pkg/operators/v1/olmconfig_types.go

```diff
@@ -0,0 +1,66 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OLMConfigSpec is the spec for an OLMConfig resource.
+type OLMConfigSpec struct {
+	Features Features `json:"features,omitempty"`
```

Comment:
Should we make this optional as well? And make it a pointer?

Reply:
Sure.

Reply:
Done!